packages/input-stepper/package.json

```diff
@@ -37,6 +37,9 @@
     "@lion/form-core": "0.6.14",
     "@lion/input": "0.10.12"
   },
+  "devDependencies": {
+    "@lion/form": "0.7.12"
```

Comment:
We can also just remove the form demo from input-stepper package so we don't need any dependency on @lion/form. This is why we have the form-integrations package anyway, to demo our fields inside a form.

Reply:
`lion-form` is used in the README.md in the using Form story

Reply:
Then we should move this story to `form-integration`, right?

Reply:
Yep! But this PR is plenty large already, I think I'll go through every package today and minimise dependencies if they are caused by form integration or validation or localisation examples. Will likely do this today and we can release this PR and mine simultaneously.